tests/unit/tags/propagation/test_binary_serializer.py

```diff
@@ -43,12 +44,9 @@ def test_from_byte_array(self):
 
     def test_to_byte_array(self):
         from opencensus.tags.tag_map import TagMap
-        from collections import OrderedDict
 
-        tag_map = OrderedDict(
-            [('key1', 'val1'), ('key2', 'val2'),
-             ('key3', 'val3'), ('key4', 'val4')])
-        tags = [tag_map]
+        tags = [Tag('key1', 'val1'), Tag('key2', 'val2'),
```

Comment:
I think `key` and `value` should be `TagKey` and `TagValue`, rather than strings?

Reply:
Can do to keep it consistent across other implementations